core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/visualization/scatterplot/ScatterplotOpDesc.scala

```diff
@@ -77,8 +93,11 @@ class ScatterplotOpDesc extends VisualizationOperator with PythonOperatorDescrip
     assert(xColumn.nonEmpty && yColumn.nonEmpty)
     val colorColExpr = if (colorColumn.nonEmpty) { s"color='$colorColumn'" }
     else { "" }
+    var argDetails = if (xLogScale) ", log_x=True" else ""
```

Comment:
I think the else statements are unnecessary. Instead, you can initialize `argDetails` with an empty string and use three different if statements. If the condition is true, append the corresponding string to `argDetails`.